template/template_example.Rmd

```diff
@@ -0,0 +1,206 @@
+
+If the concept of a "file path" is unfamiliar to you; we recommend taking a look at our [section about file paths](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#an-important-note-about-file-paths-and-Rmds). 
+
+# Using a different refine.bio dataset with this analysis?
+
+If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend placing the files in the `data/` directory you created and changing the filenames and paths in the notebook to match these files (we've put comments to signify where you would need to change the code).
+We suggest saving plots and results to `plots/` and `results/` directories, respectively, as these are automatically created by the notebook.
+From here you can customize this analysis example to fit your own scientific questions and preferences. 
+
+***
+
+# <<NAME OF example>>
+
+## Install libraries
+
+See our getting started with [package installation](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#what-you-need-to-install) for a list of the other software you will need, as well as more tips and resources. 
+
+In this analysis, we will be using <<PACKAGE>> [<<CITATION>>].
+
+```{r}
+if (!(<<PACKAGE>> %in% installed.packages())) {
+  # Install this package if it isn't installed yet
+  BiocManager::install(<<PACKAGE>>, update = FALSE)
+}
+```
+
+Attach the <<PACKAGE>>:
+
+```{r}
+# Attach the library
+library(<<PACKAGE>>)
+
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+<<SET THE SEED IF YOU NEED>>
+```
+
+## Import and set up data
+
+Data downloaded from refine.bio include a metadata tab separated values ("tsv") file and a data tsv file. This chunk of code will read the both tsv files and add them as data.frames to your environment.
+
+```{r Import data from .tsv files}
+# Read in metadata tsv file
+metadata <- readr::read_tsv(file.path(data_dir, # Replace with path to your metadata file
+                                      <<METADATA_ACCESSION FILENAME>> # Replace with the name of your metadata file
+                                      ))
+
+# Read in data tsv file
+df <- readr::read_tsv(file.path(data_dir, # Replace with path to your data file
+                                <<DATA_ACCESSION FILENAME>> # Replace with the name of your data file
+                                )) %>%
+  tibble::column_to_rownames('Gene')
+```
+
+Let's ensure that the metadata and data are in the same sample order. 
+
+```{r}
+# Make the data in the order of the metadata
+df <- df %>% dplyr::select(metadata$geo_accession)
+
+# Check if this is in the same order
+all.equal(colnames(df), metadata$geo_accession)
+```
+
+
+<<<< ALL THE ANALYSIS STEPS HERE>>>>>>>>
+
+
+# Further learning resources about this analysis
+
+- [<<HELPFUL RESOURCE NAME>>](<<LINK TO HELPFUL RESOURCE>>) [<<CITATION>>]
+
+## Print session info:
```

Comment:
Semantically, this should be an h1 (it is not a subsection of Further learning)? Or here, I might actually go against my general preference and just use a bold face...

Reply:
I think its nice for it to be in the TOC.